### ISSUE TITLE
Dockerfile

### ISSUE THREAD

**Issue by moqmar** [robinp7720/Oblecto#6]
Is there any plan to add Docker support?

I can write a Dockerfile for Oblecto and open a pull request, would just like to know if that's something that you want to be officially supported. I'd really prefer an official image instead of some random one.

**Comment by robinp7720** [comment]
Yes! Docker support would be greatly appreciated. I don't have much experience with it myself but would love a pull request which adds it.

**Comment by moqmar** [comment]
I'm currently stuck at https://github.com/agnat/node_mdns/issues/92; it seems like I'll also have to add the Avahi Daemon.

**Comment by gerroon** [comment]
Yes docker please, both for the server and the browser possibly?

**Comment by robinp7720** [comment]
Looks excellent!
The user management commands should be easy enoughl. User management only needs to modify the users table. Passwords are hashed using bcrypt. This could be done be calling the REST endpoints for those actions or a separate shell script which interacts with the MySQL database. 

The Oblecto-Web problem is slightly more difficult. The host config option is use only once, that is in main.js to set the host in the vuex store. I think having the Oblecto server itself host it under /web and assuming that the Oblecto backend has the same hostname where the frontend is being served from using the window.location.hostname value would be the easiest solution.

**Comment by robinp7720** [comment]
Ok, I've added the Oblecto-Web repo as a git submobule so a recursive git clone will pull Oblecto-Web 
as a sub directory to the Oblecto server. Oblecto then exposes the built Oblecto-Web SPA under /web.

I've added a build-web command to the Oblecto backend package.json which will build the Oblecto-Web frontend with all the correct paths and base urls.

This should make the docker container setup a bit easier since you'll only need two containers now instead of three.
